### ISSUE TITLE
Vulnerability (#25) Have problem !

### ISSUE THREAD

**Issue by Tinky124** [propstg/esx-blarglebus#27]
When i have busdriver then it kicks me out from the server ! :) Because you use if is my job busdriver then player.kick from the server ! :D You need to fix that

**Comment by propstg** [comment]
@Tinky124 I replied to your [commit comment](https://github.com/propstg/esx-blarglebus/commit/ede981cd00a85737d1c707739dadd99781296f8d), but I'll paste it here as well, since the questions are still relevant.
> Hmm, I didn't see that behavior when testing it last night, but I'll test it some more, soon.
>
> You tested it on your server and it kicked you out when you were busdriver, though? Or is this just a visual inspection? because ~= means not equal, so it should be read as 'so if player does not have job then kick him out"

As a follow-up to that:  I retested this over my lunch break and I wasn't able to recreate. I added log statements to show the result of the logic, but it's otherwise the same as master.

I took the busdriver job, triggered the two events that make money, switched to ambulance, and ran a script to fire off the finshRoute event. I didn't get kicked when I was busdriver, but did when I was ambulance.

![esx-blarglebus-27](https://user-images.githubusercontent.com/22607691/74764984-be318880-5250-11ea-8133-5b809b3a8afb.png)

**Comment by propstg** [comment]
Alright, I just finished adding [unit tests](https://github.com/propstg/esx-blarglebus/pull/28/files#diff-8f84e5f44344a88fbff05d5e0ddb7823) for the server side script, to confirm it's working as expected. And then retested the exploit code (by completing a full stock route, in case I broke anything fixing linting errors, since none of the client code has unit tests)... Still unable to recreate.

So, I guess... what steps do I need to take to recreate the behavior that you're reporting?
